prod/native/building/conan/profiles/linux-x86-64-release

```diff
@@ -17,3 +17,4 @@ CC=/opt/gcc-14.2.0/bin/gcc
 CXX=/opt/gcc-14.2.0/bin/g++
 LDFLAGS+=-static-libgcc
 LDFLAGS+=-static-libstdc++
+CC_FOR_BUILD=/opt/gcc-14.2.0/bin/gcc
```

Comment:
This is just for package php-headers-8* package build purpose. Flex (which is required by bison) recipe is broken and it is unable to detect gcc automatically. It is not required (bison and flex as well) during agent build